### PR TITLE
feat: add Storybook stories for bridge components

### LINKED ISSUE #20: Add Storybook stories for bridge components
### Description
Create Storybook stories for: BridgeCard, AmountInput, TokenSelector, FeeBreakdown, FeeEstimate.

### Acceptance Criteria
- [ ] Stories for all bridge components
- [ ] Interaction tests for decimal validation
- [ ] MAX button behavior tests
- [ ] Gas buffer interaction tests
- [ ] Loading states coverage

### PR DESCRIPTION
**Pull request by rube-de**
## Summary

Adds comprehensive Storybook stories for 5 bridge UI components:

- **BridgeCard** (5 stories): Container with sections and divider variants
- **AmountInput** (14 stories): Decimal validation, MAX button, gas buffer interaction tests
- **TokenSelector** (12 stories): Dropdown interaction, keyboard navigation accessibility
- **FeeBreakdown** (9 stories): Loading states, partial loading, collapsed view
- **FeeEstimate** (9 stories): Responsive layout, loading states

### Interaction Tests Included

| Test | Coverage |
|------|----------|
| Decimal validation | Reject excess precision, invalid characters |
| MAX button | Click fills maxValue, respects gas buffer |
| Gas buffer | Native tokens subtract buffer, ERC20 uses full amount |
| Edge cases | Zero balance, maxValue < gasBuffer |
| Keyboard a11y | Enter/Space open, Arrow navigate, Escape close |

## Test Plan

- [x] TypeScript check passes (`yarn checkTs`)
- [x] ESLint passes (`yarn lint`)
- [x] Storybook build succeeds (`yarn storybook build`)
- [ ] Manual verification in Storybook UI

Fixes #20

### CODE REVIEW

File: frontend/src/stories/Bridge/AmountInput.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+  },
+}
+
+export const Disabled: Story = {
+  args: {
+    value: '100',
+    label: 'Amount',
+    disabled: true,
+    balance: parseUnits('500', 18),
+  },
```
[the remark this answers is not in the document]
Comment (rube-de) [reply frontend/src/stories/Bridge/AmountInput.stories.tsx]: Good catch! Added `maxValue` to the Disabled story args so the MAX button is rendered and can be tested. Fixed in 6979157.
